### DISCUSSION TITLE
Command giving me "Quote>"

### DISCUSSION THREAD

**Discussion by marlouis** [openai/whisper#1202] [Q&A] [answered]
Hi, 

I'm getting "Quote>" line when pasting the whisper command. Any idea about what I'm doing wrong ? 

`josephdupont@MacBook-Pro-de-Joseph ~ % pip3 install -U openai-whisper 
Defaulting to user installation because normal site-packages is not writeable
Requirement already satisfied: openai-whisper in ./Library/Python/3.9/lib/python/site-packages (20230314)
Requirement already satisfied: ffmpeg-python==0.2.0 in ./Library/Python/3.9/lib/python/site-packages (from openai-whisper) (0.2.0)
Requirement already satisfied: numba in ./Library/Python/3.9/lib/python/site-packages (from openai-whisper) (0.56.4)
Requirement already satisfied: torch in ./Library/Python/3.9/lib/python/site-packages (from openai-whisper) (2.0.0)
Requirement already satisfied: more-itertools in ./Library/Python/3.9/lib/python/site-packages (from openai-whisper) (9.1.0)
Requirement already satisfied: numpy in ./Library/Python/3.9/lib/python/site-packages (from openai-whisper) (1.23.5)
Requirement already satisfied: tqdm in ./Library/Python/3.9/lib/python/site-packages (from openai-whisper) (4.65.0)
Requirement already satisfied: tiktoken==0.3.1 in ./Library/Python/3.9/lib/python/site-packages (from openai-whisper) (0.3.1)
Requirement already satisfied: future in /Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/site-packages (from ffmpeg-python==0.2.0->openai-whisper) (0.18.2)
Requirement already satisfied: requests>=2.26.0 in ./Library/Python/3.9/lib/python/site-packages (from tiktoken==0.3.1->openai-whisper) (2.28.2)
Requirement already satisfied: regex>=2022.1.18 in ./Library/Python/3.9/lib/python/site-packages (from tiktoken==0.3.1->openai-whisper) (2023.3.23)
Requirement already satisfied: setuptools in /Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/site-packages (from numba->openai-whisper) (58.0.4)
Requirement already satisfied: llvmlite<0.40,>=0.39.0dev0 in ./Library/Python/3.9/lib/python/site-packages (from numba->openai-whisper) (0.39.1)
Requirement already satisfied: typing-extensions in ./Library/Python/3.9/lib/python/site-packages (from torch->openai-whisper) (4.5.0)
Requirement already satisfied: sympy in ./Library/Python/3.9/lib/python/site-packages (from torch->openai-whisper) (1.11.1)
Requirement already satisfied: filelock in ./Library/Python/3.9/lib/python/site-packages (from torch->openai-whisper) (3.10.7)
Requirement already satisfied: jinja2 in ./Library/Python/3.9/lib/python/site-packages (from torch->openai-whisper) (3.1.2)
Requirement already satisfied: networkx in ./Library/Python/3.9/lib/python/site-packages (from torch->openai-whisper) (3.1)
Requirement already satisfied: charset-normalizer<4,>=2 in ./Library/Python/3.9/lib/python/site-packages (from requests>=2.26.0->tiktoken==0.3.1->openai-whisper) (3.1.0)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in ./Library/Python/3.9/lib/python/site-packages (from requests>=2.26.0->tiktoken==0.3.1->openai-whisper) (1.26.15)
Requirement already satisfied: certifi>=2017.4.17 in ./Library/Python/3.9/lib/python/site-packages (from requests>=2.26.0->tiktoken==0.3.1->openai-whisper) (2022.12.7)
Requirement already satisfied: idna<4,>=2.5 in ./Library/Python/3.9/lib/python/site-packages (from requests>=2.26.0->tiktoken==0.3.1->openai-whisper) (3.4)
Requirement already satisfied: MarkupSafe>=2.0 in ./Library/Python/3.9/lib/python/site-packages (from jinja2->torch->openai-whisper) (2.1.2)
Requirement already satisfied: mpmath>=0.19 in ./Library/Python/3.9/lib/python/site-packages (from sympy->torch->openai-whisper) (1.3.0)`

Then :

`josephdupont@MacBook-Pro-de-Joseph ~ % whisper /Users/josephdupont/Library/Mobile Documents/com~apple~CloudDocs/Documents/Prep'ENA 2023/Tristan cours droit public/Enregistrements Tristan/TEST.mp3 --language French       
quote> 
`

Regards,

**Comment by ryanheise** [comment, marked as answer]
This part: `Prep'ENA` contains a quote. The command line interface thinks you're opening a quote and is waiting for you to close the quote by typing another `'`.

If your folder name has an actual quote character in it, then you have to type the folder name with a `\'` instead of a `'`. So, `Prep\'ENA`.
